src/common/utils/translation/getJourneyChildrenTranslationKey.ts

```diff
@@ -1,14 +1,14 @@
 import { JourneyTypeName } from '../../../domain/challenge/JourneyTypeName';
 import TranslationKey from '../../../types/TranslationKey';
 
-const getJourneyChildrenTranslationKey = (type: JourneyTypeName): TranslationKey => {
+const getJourneyChildrenTranslationKey = (type: JourneyTypeName, plural: boolean = true): TranslationKey => {
   switch (type) {
     case 'space':
-      return 'common.challenges';
+      return plural ? 'common.challenges' : 'common.challenge';
```

Comment:
we need to pass the count as far as I'm seeing in the docs. I'm going to try something

Reply:
please take a look to my last commit.
I am not moving those `common.challenges/opps...` because those are used everywhere... but I think this is cleaner now